src/features/FeatureTypes.ts

```diff
@@ -0,0 +1,28 @@
+export enum Severity {
+    Error = "error",
+    Deprecation = "deprecation",
+    Warning = "warning",
+    Info = "info",
+}
+
+export enum Stage {
+    PreBuild = "pre-build",
+    PostBuild = "post-build",
+}
+
+const DEFAULT = 1;
```

Comment:
replace it with enums:
    export const enum VisualUpdateType {
        Default= 1 << 1,
        Slicer = 1 << 2,
        Matrix = 1 << 3,
        All = Default | Slicer | Matrix
    }